postgres/datadog_checks/postgres/postgres.py

```diff
@@ -669,6 +670,8 @@ def _collect_stats(self, instance_tags):
         if self._config.collect_count_metrics:
             # Count metrics are collected from all databases discovered
             per_database_metric_scope.append(COUNT_METRICS)
+        if self._config.collect_checksum_metrics and self.version >= V12:
+            per_database_metric_scope.append(CHECKSUM_METRICS)
```

Comment:
Why do we need to collect `CHECKSUM_METRICS` here while you already added `NEWER_12_METRICS` in https://github.com/DataDog/integrations-core/pull/17203/files#diff-da451a17653eec4cd61aa1f5a4012912d1a4795d78ee4ef29abb1ff81d4aa73bR83-R84

Reply:
Earlier attempt at tests, cleaning it up